src/pages/en/guides/deploy/deno.md

```diff
@@ -7,48 +7,159 @@ i18nReady: true
 
 You can deploy a server-side rendered Astro site to [Deno Deploy](https://deno.com/deploy), a distributed system that runs JavaScript, TypeScript, and WebAssembly at the edge, worldwide.
 
-## How to deploy
+This guide includes instructions for deploying to Deno Deploy through Github Actions or Deno Deploy's CLI.
+
+## Requirements
+
+This guide assumes you already have [Deno](https://deno.land/) installed.
+
+## Project Configuration
+
+Your Astro project can be deployed to [Deno Deploy](https://deno.com/deploy) as a server-side rendered site (SSR). Deno Deploy does not support static sites.
+
+### Adapter for SSR
 
-Deploy an Astro site on Deno using the Deno adapter & [deployctl](https://github.com/denoland/deployctl).
+To enable SSR in your Astro project and deploy on Deno Deploy:
 
-1. Install Deno Adapter in Astro
+1. Install [the Deno adapter](https://github.com/withastro/astro/tree/main/packages/integrations/deno) to your project’s dependencies.
 
-    Install the Deno (beta) adapter with `npm i -D @astrojs/deno`.
+    ```bash
+      npm install @astrojs/deno
+    ```
 
-    Change your `astro.config.mjs` configuration file to the following:
+1. Add two new lines to your `astro.config.mjs` project configuration file.
```

Comment:
@harrytruong This says two new lines, but I only see one `+` in the diff below. And, I think both the `output` and the `adapter` will be necessary, so maybe all three of them should be marked as new lines here? Can you please verify this and make sure the wording is consistent with the code sample?

Reply:
thanks for catching this! yeah, should be three new lines here, will fix.